Tests/Particles/GhostsAndVirtuals/main.cpp

```diff
@@ -40,7 +40,7 @@ void test_ghosts_and_virtuals (TestParams& parms)
     const Box domain(domain_lo, domain_hi);
 
     // Define the refinement ratio
-    Vector<int> rr(nlevs-1);
+    Vector<int> rr(nlevs);
     for (int lev = 1; lev < nlevs; lev++)
         rr[lev-1] = 2;
```

Comment:
Since this is not performance-critical code, you can access with bounds-checked methods here:
```suggestion
        rr.at(lev-1) = 2;
```